frontend/src/App.js

```diff
@@ -48,15 +52,38 @@ function App() {
 
   const renderAuthenticatedRoutes = () => (
     <>
-      <Route Home path="/" component={Home} />
-      <Route path="/admin/:userId?" component={Admin} />
+      <Route
+        path="/"
+        render={() => (
+          <Page title="TTA Smart Hub">
```

Comment:
The title for this page doesn't seem to go back to `TTA Smart Hub` after navigating away and back to the Home page.

❤️  that each page has a title set, though

Reply:
Should be working now